app/views/tags/edit.html.erb

```diff
@@ -18,7 +18,7 @@
       <div class="govuk-grid-column-two-thirds" id="<%= tag_field.id %>">
         <%= render "tags/tags/#{tag_field.type}_input",
           tag_field: tag_field,
-          tags: @revision.tags,
+          tags: @edition.tags,
```

Comment:
Was this meant to be in this commit? 

Anywho I believe it should still be `@revision` as that reflects the users input if validation fails whereas edition would always be the saved version.

Reply:
Ah true - I didn't consider the 'update' action.

Reply:
Reverted it back.